test/built-ins/Array/prototype/flatMap/thisArg-argument.js

```diff
@@ -0,0 +1,20 @@
+// Copyright (C) 2018 Shilpi Jain. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+esid: sec-array.prototype.flatMap
+description: >
+    Behavior when thisArg is provided
+    Array.prototype.flatMap ( mapperFunction [ , thisArg ] )
+includes: [compareArray.js]
+---*/
+
+"use strict";
```

Comment:
Remove this line. If this test must only be executed in strict mode, then add the appropriate frontmatter: `flags: [onlyStrict]`